base/server/python/pki/server/deployment/scriptlets/configuration.py

```diff
@@ -1091,6 +1093,8 @@ def spawn(self, deployer):
                 self.configure_system_certs(deployer, subsystem)
                 self.update_system_certs(deployer, nssdb, subsystem)
                 subsystem.save()
+                print "cfu: sleep for 30 seconds before validate_system_certs() call"
+                time.sleep(30);
```

Comment:
You need to `import time` at the top of the script otherwise you will get the exception `NameError: name 'time' is not defined`.